parlai/crowdsourcing/tasks/pairwise_per_turn_eval/task_config/task_description.html

```diff
@@ -0,0 +1,34 @@
+<h1>Pairwise Dialogue Evaluations for Chatbots</h1>
```

Comment:
Honestly I'd just replace this with the Lorem Ipsum text from `parlai/crowdsourcing/tasks/model_chat/task_config/task_description.html` - we have a lot of legalese in here that wouldn't be used by end users